### PR TITLE
Moving FAQ tab from AboutUs to What is Health Equity

### PR DESCRIPTION
**Pull request by kkatzen**
Addresses part of #620

### CODE REVIEW

File: frontend/src/pages/DataCatalog/DataCatalogTab.tsx [hunk 1]
```diff
@@ -33,7 +33,6 @@ function DataCatalogTab() {
         indicatorColor="primary"
         textColor="primary"
         centered
-        style={{ marginTop: "40px" }}
```
Comment (vanshkumar) [inline frontend/src/pages/DataCatalog/DataCatalogTab.tsx:36]: What's the reason for removing this? seems unrelated to the PR title

**Review by vanshkumar — APPROVED** [review]

File: frontend/src/pages/DataCatalog/DataCatalogTab.tsx (hunk 1 above)
Comment (kkatzen) [reply frontend/src/pages/DataCatalog/DataCatalogTab.tsx:36]: I applied this style to all Tabs in MaterialTheme.tsx so it wasn't needed here any longer

**Review by efregoso — APPROVED** [review]
